pages/en/lb3/datasources.json.md

```diff
@@ -33,6 +33,19 @@ For example:
   }
 }
 ```
+To see data stored in memory connector, add <code> "file": "<filename>"</code>
```

Comment:
When i'm viewing the file (from clicking view), `"<filename>"` is gone.  You might wanna take a look. 

Suggested change: "To store data in a file for memory connector, add...."